peps/pep-9999.rst

```diff
@@ -0,0 +1,546 @@
+PEP: 9999
+Title: Classes & protocols: Read-only attributes
+Author: <REQUIRED: list of authors' real names and optionally, email addrs>
```

Comment:
I guess we forgot to update the "real names" part here. It's fine to say something like "Enegg <your@email.com>" (#4058). I'll look into updating the template.